x-pack/plugins/alerting/server/task_runner/task_runner.ts

```diff
@@ -262,44 +262,55 @@ export class TaskRunner<
 
     let updatedAlertTypeState: void | Record<string, unknown>;
     try {
-      updatedAlertTypeState = await this.alertType.executor({
-        alertId,
-        services: {
-          ...services,
-          alertInstanceFactory: createAlertInstanceFactory<
-            InstanceState,
-            InstanceContext,
-            WithoutReservedActionGroups<ActionGroupIds, RecoveryActionGroupId>
-          >(alertInstances),
-        },
-        params,
-        state: alertTypeState as State,
-        startedAt: this.taskInstance.startedAt!,
-        previousStartedAt: previousStartedAt ? new Date(previousStartedAt) : null,
-        spaceId,
-        namespace,
-        name,
-        tags,
-        createdBy,
-        updatedBy,
-        rule: {
+      const ctx = {
+        type: 'alert',
```

Comment:
Is this `alert` to align with the SO type `alert`? If not, could we make it `rule` to follow our new terminology?

Reply:
Or maybe `alerting` to align with the plugin name?

Reply:
dang it!  I'm pretty good at catching "alert" when I write now, but apparently not good at catching when I read :-)

I'll open an issue - I was thinking we might want to tweak some of those messages anyway, but didn't want to have that delay the merge ...

Reply:
opened issue https://github.com/elastic/kibana/issues/108776 to track updating the wording in the context variables